### PR TITLE
Style fix

### LINKED ISSUE #4: Coding style?
I really like google javascript style. if you agree, I would send PR to make all code as google javascript style.

### PR DESCRIPTION
**Pull request by sunglim**
- Moving extension codes to /extension
- Adding package.json and Gruntfile.js to run jshint and jscsrc. they will check error and style.
- Style fix
- Adding CONTRIBUTING.md to describe how to setup.

Fixes https://github.com/yperess/gerrit-chrome-extension/issues/4

@yperess

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
+*.swp
```
Comment (yperess) [inline .gitignore]: Should we really have any .swp files layout around? This seems too platform dependent.

**Review by yperess — CHANGES_REQUESTED** [review]
Small nit, but overall LGTM. Please update .gitignore and I'll merge this :)

File: .gitignore (hunk 1 above)
Comment (sunglim) [reply .gitignore]: Done.

**Comment by sunglim** [comment]
PTAL.

**Review by yperess — APPROVED** [review]
+2

**Comment by sunglim** [comment]
Thanks!

**Pull request merged**